pkg/k8s/client.go

```diff
@@ -2,8 +2,12 @@ package k8s
 
 import (
 	"context"
+	"errors"
+	"fmt"
 	"strings"
+	"time"
 
+	"github.com/jellydator/ttlcache/v3"
```

Comment:
Why do we need external lib/dep for caching?

Reply:
I have replaced it with go-map.